FinancialModelingPrepApi/Core/Crypto/CryptoMarketProvider.cs

```diff
@@ -28,10 +28,23 @@ public Task<ApiResponse<List<CryptoItem>>> GetAvilableCryptocurrencies()
                 { "version", ApiVersion.v3.ToString() }
             };
 
-            return client.GetJsonAsync<List<CryptoItem>>(url, pathParams, null);
+            return await client.GetJsonAsync<List<CryptoItem>>(url, pathParams, null);
         }
 
-        public Task<ApiResponse<CryptoHistoricalPriceDailyItem>> GetDailyPrices(string symbol)
+        public async Task<ApiResponse<List<CryptoQuoteResponse>>> GetQuoteAsync(string symbol)
```

Comment:
If you just await the Task without doing anything with it you can remove the async/await keywords here. 
Only the first caller in the call stack that needs to do something with the result needs to await it.